src/App.js

```diff
@@ -3,19 +3,20 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import List from './components/List';
```

Comment:
I see some warnings that this **List** is defined but not used

Reply:
I think we forgot to remove it when we removed the list route in that file. We'll remove it when we merge. :) 
Thank you so much for the review. ❤️